overlays/profiles/motorbike.vspec

```diff
@@ -42,4 +43,63 @@ Vehicle.Cabin.Handlebar.GripHeating:
   type: sensor
   datatype: string
   allowed: ["HEATING_OFF", "HEATING_STEP_1", "HEATING_STEP_2", "HEATING_STEP_3", "HEATING_STEP_4", "HEATING_STEP_5", "HEATING_AUTOMATIC"]
-  description: Grip Heating Status in motorbikes. The signal shows if the handlebar grip heating is on or not, and if yes, to which level.
\ No newline at end of file
+  description: Grip Heating Status in motorbikes. The signal shows if the handlebar grip heating is on or not, and if yes, to which level.
+
+Vehicle.LeanAngle:
+    type: sensor
+    datatype: float
+    unit: degrees
+    description: Lean Angle of the Motorcycle in Y direction. 0 = upright, 
+                 positive = leaning right from rider perspective,
+                 negative = leaning left from rider perspective.
+
+Vehicle.IMU:
+    type: branch
+    description: Information related to the Inertial Measurement Unit to record angular rates and velocity-related information.
+
+Vehicle.IMU.IsOK:
+    type: sensor
+    datatype: boolean
+    description: Indicates if IMU is working. True = OK. False = Failure.
+
+# Next branch and signal can be removed when https://github.com/COVESA/vehicle_signal_specification/pull/478 is merged
+Vehicle.LowVoltageBattery:
+    type: branch
+    description: SLI Battery (Start, Light, Ignition)
+
+Vehicle.LowVoltageBattery.CurrentVoltage:
+    type: sensor
+    datatype: float
+    unit: V
+    description: actual voltage of the SLI Battery
+
+Vehicle.ADAS:
+    type: branch
+
+# Motorbikes typically have different ABS-systems for front and rear wheel
+Vehicle.ADAS.ABS:
+    type: branch
+    instances: ["Front","Rear"]
+
+Vehicle.Powertrain:
+    type: branch
+
+Vehicle.Powertrain.Transmission:
+    type: branch
+
+Vehicle.Powertrain.Transmission.IsClutchSwitchOperated:
+    type: sensor
+    datatype: boolean
+    description: Indicates if the Clutch switch is operated, so engine and transmission are decoupled. False = Clutch switch not operated. True = Clutch switch operated.
+    comment: True corresponds to that Vehicle.Powertrain.Transmission.ClutchEngagement is 0.0 or very close to 0.0.
```

Comment:
Some minor change here - I changed the signal name to comply with VSS "Is*" naming recommendation. But then It got a bit unsure on what we mean with this signal vs. Vehicle.Powertrain.Transmission.ClutchEngagement. Shall this be true as soon as you barely move the clutch switch, i.e. corresponding to ClutchEngagement is just below 100%, or shall it be true first when ClutchEngagement is almost 0?